README.md

```diff
@@ -1,18 +1,24 @@
-IPFS webui
-==========
+# IPFS webui
 
 [![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io) [![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/) [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs) [![Dependency Status](https://david-dm.org/ipfs/webui.svg?style=flat-square)](https://david-dm.org/ipfs/webui) [![Circle CI](https://img.shields.io/circleci/project/ipfs/webui/master.svg?style=flat-square)](https://circleci.com/gh/ipfs/webui) [![Travis CI](https://img.shields.io/travis/ipfs/webui/master.svg?style=flat-square)](https://travis-ci.org/ipfs/webui)
 
-> IPFS Web Interaface
+> The web interface for [IPFS](https://ipfs.io/)
 
-## Dev
+## Development
+
+Make sure [node.js](https://nodejs.org/) and [npm](https://docs.npmjs.com/) are installed and in your path.
 
 ```bash
-git clone https://github.com/ipfs/webui
-cd webui
-npm install
-# Runs server on port 3000. Set env PORT to change.
-npm run dev
+$ git clone https://github.com/ipfs/webui
+$ cd webui
```

Comment:
These two lines are kind of unnecessary. I renamed mine to `ifps-webui`, and I suspect others do the same.